### DISCUSSION TITLE
Build queue solution?

### DISCUSSION THREAD

**Discussion by birkasecorba** [vercel/community#4254] [Help] [answered]
### Summary

It seems like they identified the issue and implemented a solution; can anyone verify? Is there another platform aside from vercel-status.com where the team provides better updates?

<img width="913" alt="image" src="https://github.com/vercel/community/assets/3323411/5e1df5a1-812e-497c-9dc1-4747331a670d">

### Steps to Reproduce

Build queue

**Comment by ofhouse** [comment, marked as answer]
We had an issue with builds for Hobby & Pro customers and are actively working on mitigation.
A fix is implemented and now our system works through all of the builds that were queued during the incident.
This can take a bit to resolve for all customers, but we're also adding additional ressources to our build system to ensure we resolve the issue as quick as possible.

We'll post on the status page once the issue is resolved: https://www.vercel-status.com/incidents/87m1ypq44hf0

**Comment by loren** [comment]
is there any indication of how long it will take to clear the queue? It seems like some builds are queued longer than the duration of the outage.

**Reply by ofhouse** [reply]
Just got the information that the queue should have been processed, do you still see stuck deployments?

**Reply by birkasecorba** [reply]
No, the issue is resolved on our end as well. Thank you.

**Reply by naegelin** [reply]
We are still seeing builds queue for 10 mins+ @ofhouse

**Reply by ofhouse** [reply]
@naegelin Thanks for the report.
Unfortunately we are still seeing some customers that are experiencing queued builds.
We'll investigate further and I made sure the team is aware of your case!
